Composer/ScriptHandler.php

```diff
@@ -459,30 +425,10 @@ protected static function getConsoleDir(CommandEvent $event, $actionName)
     {
         $options = static::getOptions($event);
 
-        if (static::useNewDirectoryStructure($options)) {
-            if (!static::hasDirectory($event, 'symfony-bin-dir', $options['symfony-bin-dir'], $actionName)) {
-                return;
-            }
-
-            return $options['symfony-bin-dir'];
-        }
-
         if (!static::hasDirectory($event, 'symfony-app-dir', $options['symfony-app-dir'], 'execute command')) {
             return;
         }
 
         return $options['symfony-app-dir'];
     }
```

Comment:
the implementation here is broken when SE 3.0 is used, as the console dir is not the app dir anymore.